pkgs/development/python-modules/imagededup/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cython
+, torch
+, torchvision
+, pillow
+, tqdm
+, scikit-learn
+, pywavelets
+, matplotlib
+}:
+
+buildPythonPackage rec {
+  pname = "imagededup";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "idealo";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-B2IuNMTZnzBi6IxrHBoMDsmIcqGQpznd/2f1XKo1Oa4=";
+  };
+
+  propagatedBuildInputs = [
+    cython
+    torch
+    torchvision
+    pillow
+    tqdm
+    scikit-learn
+    pywavelets
+    matplotlib
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = "https://idealo.github.io/imagededup/";
```

```suggestion
    homepage = "https://idealo.github.io/imagededup/";
    changelog = "https://github.com/idealo/imagededup/releases/tag/${src.rev}";
```
Adding meta.changelog is helpful for future updates.